wcbench.sh

```diff
@@ -925,6 +1039,14 @@ while getopts ":hvrcip:ot:kd" opt; do
             uninstall_cbench
             action_taken=true
             ;;
+        u)
```

Comment:
We should set a var here, like `OS="ubuntu"`, then check it in the places that I mentioned OS-specific conditionals above.